### PR TITLE
Feature/Refactor Fuel Information Related Vehicle Data

### PR DESCRIPTION
**Pull request by YarikMamykin**
Fixes [#3124](https://github.com/smartdevicelink/sdl_core/issues/3124)

This PR is **ready** for review.

### Testing Plan
Manual testing

### Summary
Updated fuelRange structure of VehicleData with new fields.

### Instruction
[Manual_checklist_instruction_Refactor_Fuel_Information_Related_Vehicle_Data.zip](https://github.com/smartdevicelink/sdl_hmi/files/4381709/Manual_checklist_instruction_Refactor_Fuel_Information_Related_Vehicle_Data.zip)

### CLA
- [X] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by LitvinenkoIra** [comment]
@atiwari9 please review this PR

**Comment by atiwari9** [comment]
PR Approved.

**Comment by LitvinenkoIra** [comment]
Resolves https://github.com/smartdevicelink/sdl_hmi/issues/261

**Comment by LitvinenkoIra** [comment]
@theresalech PR is ready for Livio review. Thank you!

File: app/model/sdl/VehicleInfoModel.js [hunk 1]
```diff
@@ -210,7 +210,11 @@ SDL.SDLVehicleInfoModel = Em.Object.create(
       'fuelRange': [
         {
           'type':'GASOLINE',
-          'range': 400
+          'range': 400,
+          'level': 10,
+          'levelState': 'NORMAL',
+          'capacity': 100,
+          'capacityUnit': 'KILOGRAMS'
```
Comment (ShobhitAd) [inline app/model/sdl/VehicleInfoModel.js:217]:
```suggestion
          'capacityUnit': 'LITERS'
```
Please refer to the `capacityUnit` param description in the `FuelRange` struct
> The unit of the capacity of this fuel type such as liters for gasoline or kWh for batteries.

**Pull request merged**